colour/utilities/common.py

```diff
@@ -302,3 +305,48 @@ def is_integer(x):
     """
 
     return abs(x - round(x)) <= INTEGER_THRESHOLD
+
+
+def filter_kwargs(function, **kwargs):
```

Comment:
It is not particularly sexy, but basically it will allow removing the `**kwargs` argument here for example: https://github.com/colour-science/colour/blob/develop/colour/difference/delta_e.py#L93 and still have the function callable in that generic one: https://github.com/colour-science/colour/blob/develop/colour/difference/delta_e.py#L403